Peer/Program.cs

```diff
@@ -215,14 +215,11 @@ public static async Task ConfigAsync(ConfigOptions _)
                 return configResults.Error;
             }
 
-            var watchOptions = configuration.GetSection("Peer")
-                .Get<WatchOptions>()
-                ?? new WatchOptions();
+            var showConfig = configuration.GetSection("Peer:Show")
```

Comment:
This is a breaking change to the config file but I think it make sense to group configuration options for each command in case multiple commands would use a similar config with the same name. I'm also good with supporting both or just including scope information in the property names themselves though.

Reply:
Lets support both for now and deprecate it for next major maybe?

Reply:
That's funny SH didn't show me your comment edits when I made my comment lol